libs/payments/capability/src/lib/capability.manager.spec.ts

```diff
@@ -5,58 +5,66 @@ import { Test, TestingModule } from '@nestjs/testing';
+  CapabilityServicesResultFactory,
   ServiceResultFactory,
   ServicesWithCapabilitiesResultUtil,
 } from '@fxa/shared/contentful';
 import { CapabilityManager } from './capability.manager';
 
 describe('CapabilityManager', () => {
-  let manager: CapabilityManager;
-  let mockContentfulManager: ContentfulManager;
-  let mockResult: ServicesWithCapabilitiesResultUtil;
-
-  beforeEach(async () => {
-    mockResult = {} as any;
-    mockContentfulManager = {
-      getServicesWithCapabilities: jest.fn().mockResolvedValueOnce(mockResult),
-    } as any;
-
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [
-        { provide: ContentfulManager, useValue: mockContentfulManager },
-        CapabilityManager,
-      ],
-    }).compile();
-
-    manager = module.get<CapabilityManager>(CapabilityManager);
-  });
+  describe('getClients', () => {
+    let manager: CapabilityManager;
+    let mockContentfulManager: ContentfulManager;
+    let mockResult: ServicesWithCapabilitiesResultUtil;
 
-  it('should be defined', async () => {
-    expect(manager).toBeDefined();
-    expect(manager).toBeInstanceOf(CapabilityManager);
-  });
+    beforeEach(async () => {
+      mockResult = {} as any;
+      mockContentfulManager = {
+        getServicesWithCapabilities: jest
+          .fn()
+          .mockResolvedValueOnce(mockResult),
+      } as any;
+
+      const module: TestingModule = await Test.createTestingModule({
+        providers: [
+          { provide: ContentfulManager, useValue: mockContentfulManager },
+          CapabilityManager,
+        ],
+      }).compile();
+
+      manager = module.get<CapabilityManager>(CapabilityManager);
+    });
+
+    it('should be defined', async () => {
+      expect(manager).toBeDefined();
+      expect(manager).toBeInstanceOf(CapabilityManager);
+    });
 
-  describe('getClients', () => {
     it('should return empty results', async () => {
       mockResult.getServices = jest.fn().mockReturnValueOnce(undefined);
       const result = await manager.getClients();
       expect(result.length).toBe(0);
     });
 
     it('should return services with capabilities', async () => {
-      const clientResults = ServiceResultFactory({
-        oauthClientId: 'client1',
-        capabilitiesCollection: {
-          items: [
-            { slug: 'exampleCap0' },
-            { slug: 'exampleCap2' },
-            { slug: 'exampleCap4' },
-            { slug: 'exampleCap5' },
-            { slug: 'exampleCap6' },
-            { slug: 'exampleCap8' },
-          ],
-        },
-      });
+      const clientResults = [
+        ServiceResultFactory({
+          oauthClientId: 'client1',
+          capabilitiesCollection: {
+            items: [
+              { slug: 'exampleCap0' },
+              { slug: 'exampleCap2' },
+              { slug: 'exampleCap4' },
+              { slug: 'exampleCap5' },
+              { slug: 'exampleCap6' },
+              { slug: 'exampleCap8' },
```

Comment:
Non-blocking since you did not add -

Might be nice for the check below (actualCapabilities [here](https://github.com/mozilla/fxa/pull/16035/files#diff-5205c29c11879e42c0601b302d9d6cb31ed12cd389b6fec4d947df8953c56bbcL65-L73)) to actually reference the `clientResults` variable rather than being hardcoded values, then wouldn't even have to specify the slugs themselves.

This should also use `CapabilitiesResultFactory`

Reply:
Updated to use `CapabilitiesResultFactory`. Updating to reference clientResults.